robot_smach_states/src/robot_smach_states/human_interaction/find_person_in_room.py

```diff
@@ -67,6 +71,10 @@ def __init__(self, robot, person_label='operator', search_timeout=60, look_dista
 
         self._room = room
 
+        self._look_angles = np.linspace(look_range[0], look_range[1], look_steps)
+
+        self.speak_when_found = speak_when_found
```

Comment:
@LoyVanBeek, Why did add the switch for speak when found?

Reply:
Because sometimes we don't want the robot to speak. Makes it seem more natural